extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -240,6 +240,14 @@ export class AmpAnalytics extends AMP.BaseElement {
     return this.iniPromise_;
   }
 
+  /**
+   * @return {boolean} whether parent post messages are allowed.
```

Comment:
Nit: maybe document our decision here. That `parentPostMessage` feature is only allowed for ads.

Reply:
Sure. Done.